view/frontend/layout/hyva_checkout_components.xml

```diff
@@ -29,6 +34,623 @@
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
+
+            <!-- Payment Renderer: MultiSafepay DotPay -->
+            <block name="checkout.payment.method.multisafepay_dotpay"
+                   as="multisafepay_dotpay">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_dotpay</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
+
+            <!-- Payment Renderer: MultiSafepay EPS -->
+            <block name="checkout.payment.method.multisafepay_eps"
+                   as="multisafepay_eps">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_eps</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
+
+            <!-- Payment Renderer: MultiSafepay GiroPay -->
+            <block name="checkout.payment.method.multisafepay_giropay"
+                   as="multisafepay_giropay">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_giropay</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
+
+            <!-- Payment Renderer: MultiSafepay Google Pay -->
+            <block name="checkout.payment.method.multisafepay_googlepay"
+                   as="multisafepay_googlepay">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_googlepay</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
+
+            <!-- Payment Renderer: MultiSafepay iDEAL -->
+            <block name="checkout.payment.method.multisafepay_ideal"
+                   as="multisafepay_ideal">
+                <arguments>
+                    <argument name="metadata" xsi:type="array">
+                        <item name="icon" xsi:type="array">
+                            <item name="svg" xsi:type="string">multisafepay/multisafepay_ideal</item>
+                        </item>
+                    </argument>
+                </arguments>
+            </block>
```

Comment:
This is duplicate and will overwrite the iDeal config block at line 13. Therefor breaking the iDeal issuer selector.

Reply:
Thanks!